### DISCUSSION TITLE
Send notification to two telegram groups

### DISCUSSION THREAD

**Discussion by gxres042** [upptime/upptime#956] [Ideas] [answered]
I want to send alerts to two telegram groups, but it seems like upptime can only specify one group's chat ID.

Any way to resolve this?

**Comment by AnandChowdhary** [comment]
Unfortunately there's no easy way to do this right now, but I think we can add this feature. For example, if you make a PR where we detect if a comma (`,`) is in the Telegram chat ID, then we split the string using the comma and make multiple requests. What say?

**Reply by gxres042** [reply]
That's great!

**Reply by gxres042** [reply, marked as answer]
https://github.com/upptime/uptime-monitor/pull/249
